molecule/user_provided/vars/vars.yml

```diff
@@ -23,3 +23,11 @@ tas_single_node_rekor_redis:
     host: "{{ ansible_default_ipv4.address }}"
     port: 6379
     password: password
+  backfill_redis:
```

Comment:
Not sure there is a reason to leave these vars here unless you change something, since they are exactly the same as in `default`. Unless you aim to change these into something that will override defaults.

Reply:
This was added here because the molecule tests were failing with this error message 
```
The conditional check 'tas_single_node_rekor_redis.backfill_redis.enabled' failed. The error was: error while evaluating conditional (tas_single_node_rekor_redis.backfill_redis.enabled): 'dict object' has no attribute 'backfill_redis'
```

I believe because backfill_redis is a property under tas_single_node_rekor_redis, it needed to be defined here aswell.

Reply:
Ah yes, I didn't see you had put the new job config under `tas_single_node_rekor_redis`